### ISSUE TITLE
Space Infiltration Design Flow Rate instances from Space Type don't show up on Spaces/AIrFlow sub-tab

### ISSUE THREAD

**Issue by DavidGoldwasser** [NREL/OpenStudio#2809]
While ventilation rates and spaces types show both on "AirFlow" and "Loads" sub tabs for space types, when you look at the Spaces tab infiltration is only on "Loads" and not "AirFlow" It is a bit odd that infiltration shows up in two different areas. Maybe it should be removed from one of the two places? But if not then what happens on Spaces, should mirror Space Types.

![screen shot 2017-10-02 at 2 22 37 pm](https://user-images.githubusercontent.com/4857055/31098552-35e701bc-a780-11e7-9fec-05f3b81f12f4.png)

![screen shot 2017-10-02 at 2 22 54 pm](https://user-images.githubusercontent.com/4857055/31098556-39887468-a780-11e7-9cea-7bd5a5a88ef8.png)

![screen shot 2017-10-02 at 2 24 29 pm](https://user-images.githubusercontent.com/4857055/31098559-3c2400a2-a780-11e7-9dc6-15d88ee5afda.png)

![screen shot 2017-10-02 at 2 25 01 pm](https://user-images.githubusercontent.com/4857055/31098563-3e26350a-a780-11e7-845f-0b18d22736ec.png)

**Comment by evanweaver** [comment]
Thanks @DavidGoldwasser for reviewing this more closely.

**Comment by DavidGoldwasser** [comment]
Below are a number of possible changes.

A) I think a minimal solution is to have inherited space infiltration objects show up in the "Properties/Airflow" section of the spaces tab. This will do two things.
1. It will be more consistent with where what you see for same objects in the space type tab
2. It will allow a modeler to see ventilation and infiltration loads in a single view (Space/Properties/Airflow) without having to go to (Space/Loads)

B) Since ventilation and infiltration were pulled out of "General" and put in "Airflow" for the spaces tab, it makes sense to add an "Airflow" tab in the Space Types tab and put it between "General" and "Loads".

C) There is also a good argument that "Airflow" on the spaces tab should be pulled out of "Properties" and be put as a new sub-tab between "Properties" and "Loads". A second part of this would be to have the name "General" in space types and "Properties" in spaces unified to same term. 

D) For both the Space Type tab and Space tab it is strange that infiltration objects show, but not ventilation objects. I'd be in support of an all or none approach (don't show infiltration, or also show ventilation). If there was just a way to add new ventilation and infiltration objects to the loads view similar to other internal loads, then there wouldn't be a need for the "Airflow" sections.

I'll make few quick mockups and update this post.

**Comment by jmarrec** [comment]
I agree with A: under the Space tab, the Airflow one should have inherited SpaceType's infiltration design flow rate, DSOA, and space infiltration effective leakage areas.

I understand the rationale of B., and it is confusing right now to put able to put a SpaceInfiltrationDesignFlowRate on the "General" tab, be able to see the object in the IG, schedule is blank, but I have to go to the Loads tab to assign a schedule... So perhaps moving it to a specific tab could give a chance to address this weird behavior.

![spacetype_general](https://user-images.githubusercontent.com/5479063/35432621-e95d9412-0280-11e8-926f-66b152475c59.png)

![spacetype_loads](https://user-images.githubusercontent.com/5479063/35432625-ed5c7a1a-0280-11e8-8112-52331f13fab6.png)

Point C: I don't really care if it gets moved to a specific tab or a subtab of General, but perhaps you're right, it'd be more consistent. (Side note: the "My Model" in the main right column controller doesn't include "Space Infiltration Design Flow Rate", I have to drag from library... true for SpaceType and Space tabs)


Point D: There's something strange regarding the ZoneVentilationDesignFlowRate. The problem is that this object is placed on ThermalZone, and one TZ can serve multiple spaces... so I'm not sure how we could show that on the space airflow tab...

**Comment by DavidGoldwasser** [comment]
@jmarrec I updated my note for D. When I said ventilation I mean Design Specification OA objects. 

I agree ZoneVentilationDesignFlowRate shouldn't be on the space tab.
